skynet.yaml

```diff
@@ -1,7 +1,7 @@
 name: verify-github-actions
 description: Verify that the github actions run passed, this is needed to make pipelines pass without manual intervention
 contact: 'Frontend Frameworks Architecture / #support-frontend-architecture'
-image: python:3.10.0
+image: drydock.workiva.net/workiva/skynet-images:pr-127
```

Comment:
Yeah I guess we most likely want to pin it to a image number don't we I was thinking it would be so easy to propagate changes through the repos that all use this image but that can be both a good and bad thing. I will pin it to an image tag.